### PR TITLE
Quick Static-text check.

### PR DESCRIPTION
**Pull request by vtange**
Went back and talked to NPCs to check their 'static', all-quests-finished lines.
Since many players will be seeing their Master's line a million times, I made sure they sound good.
**Already good**
Ranger
Archer
Cleric
Krivis
Swordsman
Cyromancer
Pyromancer
Peltasta

**Worked on:**
Corsair
Pardoner
Elementalist
Wizard
Bokor
Sadhu
Highlander
Quarrel Shooter
Priest Master
Monk Master
Hoplite Master
Hunter Master
Sapper Master
Alchemist Master? (Vaidotas)
Paladin Master
Standardized -> Capitalized "Watcher" position, as it seems like a group name.
Standardized -> Capitalized "Believer" position, as it seems like a group name.
**Maps checked (for personal ref)**
West Siauliai
Klaipeda
East Siauliai
Miner's village
Srautas
Gele
Nefritas
Tenet Gard
Tenet Chap
Veja -> Septyni (very few npcs)
Gate Route
Sirdgela
Kvailas
Dvasia
Great King
Ramstis Ridge
Overlong
Akmen
Tiltas Valley
Rukas Plateau
Gytis
Baron Allerno
Myrkiti
Aqueduct
Greene

### CODE REVIEW

File: QUEST.tsv [hunk 1]
```diff
@@ -565,8 +565,8 @@ QUEST_20150317_000564	Compass
 QUEST_20150317_000565	Select Information
 QUEST_20150317_000566	Warning
 QUEST_20150317_000567	This is the entrance to Kateen Forest. {nl}Travelling is ill-advised because of the vicious monsters inhabiting the area.
-QUEST_20150317_000568	We are carrying out the plan to get through the Gate Route. {nl}Dangerous monsters are showing up, so travelers should be careful. {nl} - Commander Julian
-QUEST_20150317_000569	There are rumors that ghosts have been appearing, so please refrain from passing through.
+QUEST_20150317_000568	$We are carrying out an operation to reach the Gate Route. {nl}Dangerous monsters are beyond here, so travellers should be careful. {nl} - Commander Julian
```
Comment (Sunaries) [inline QUEST.tsv]: We're using US English, so it should be travelers instead of travellers.

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -539,8 +539,8 @@ QUEST_LV_0100_20150317_000534	It's important to keep in mind what you would want
 QUEST_LV_0100_20150317_000535	{memo X}Yes. That's how you do it. {nl} Pretty simple, huh?
 QUEST_LV_0100_20150317_000536	Aras is in the center area of the woods. {nl} But there has been a tremendous increase in monsters around here, so I suggest you don't go in.
 QUEST_LV_0100_20150317_000537	Somehow Klaipeda has managed to hold back the monsters for now,{nl}but we have to find and remove the cause or we won't hold out for long.{nl}
-QUEST_LV_0100_20150317_000538	That strange feeling I had in Crystal Mine 3F,{nl}it keeps occurring to me that it might be related to that girl the town people are talking about.{nl}
-QUEST_LV_0100_20150317_000539	I just feel bad that I wasn't able to meet
+QUEST_LV_0100_20150317_000538	$That strange feeling I had at the 3rd floor of the Crystal Mines,{nl}it keeps occurring to me that it might be related to that girl the townspeople keep talking about.{nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: I believe most of the lines (dialogue) are standardized to "X 1F" etc.

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -942,7 +942,7 @@ QUEST_LV_0100_20150317_000937	It's amazing you can use the barrier like that. {n
 QUEST_LV_0100_20150317_000938	{memo X}
 QUEST_LV_0100_20150317_000939	Holding off against this many enemies is by itself a great feat. {nl} It's all thanks to the Paladin Master.
 QUEST_LV_0100_20150317_000940	Even though he's still young, Kenneth is amazing. {nl}I think we can trust him a little more.
-QUEST_LV_0100_20150317_000941	I have received an urgent message from the watchers.{nl}A gigantic demon monster is coming this way.{nl}Quickly, follow me.{nl}
+QUEST_LV_0100_20150317_000941	$I have received an urgent message from the Watchers.{nl}A gigantic demon monster is coming this way.{nl}Quickly, follow me.{nl}
 QUEST_LV_0100_20150317_000942	Biteregina, which used to live in the deep forest, has appeared here.{nl}It even built a hive at the upper side of Rojus Plateau. Since it's dangerous, can you defeat it?
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: I

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -2161,30 +2161,30 @@ QUEST_LV_0100_20150317_002156	God is looking after our village so our village ha
...
-QUEST_LV_0100_20150317_002169	The fact that we were able meet was through God's guidance.
-QUEST_LV_0100_20150317_002170	Please refrain from thinking about all the suffering. We have all received a blessing.
-QUEST_LV_0100_20150317_002171	Trust the god.{nl}We can be together in the tree.
-QUEST_LV_0100_20150317_002172	We will be saved by being together.{nl}The god had told us to do that.
+QUEST_LV_0100_20150317_002167	$It is a good sign that you, the Revelator, came here.{nl}We've been waiting for you for a long time.
+QUEST_LV_0100_20150317_002168	$Please come to our village.{nl}The villagers will really like you.{nl}For sure.
+QUEST_LV_0100_20150317_002169	$The fact that we were able meet was through the Goddess' guidance.
+QUEST_LV_0100_20150317_002170	$Please refrain from thinking about any suffering. We have all received a blessing.
+QUEST_LV_0100_20150317_002171	Trust the Goddess.{nl}We can be together in the tree.
+QUEST_LV_0100_20150317_002172	We will be saved by being together.{nl}The goddess had told us to do that.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: goddess -> Goddess

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -2161,30 +2161,30 @@ QUEST_LV_0100_20150317_002156	God is looking after our village so our village ha
...
 QUEST_LV_0100_20150317_002173	Not yet?{nl}Where is the girl?
 QUEST_LV_0100_20150317_002174	{memo X}I can't use my power since I was deceivied by the demon's trick.{nl}You should find my power to guide you to the revelation.
-QUEST_LV_0100_20150317_002175	That portal is the gate to lies.{nl}Laima has brought you to me because of that.
-QUEST_LV_0100_20150317_002176	Demon Lord Bramble can absolutely never be forgiven.
-QUEST_LV_0100_20150317_002177	Bramble must be recovering his power even at this moment.{nl}Please meet the believers at Gate Route.
-QUEST_LV_0100_20150317_002178	Please be careful of the trick of the observer at Gate Route.
+QUEST_LV_0100_20150317_002175	$That portal is a gate to lies.{nl}Laima has brought you to me because of that.
+QUEST_LV_0100_20150317_002176	$Demon Lord Bramble absolutely can never be forgiven.
+QUEST_LV_0100_20150317_002177	$Bramble must be recovering his power even at this moment.{nl}Please meet the believers at Gate Route.
+QUEST_LV_0100_20150317_002178	$Please be careful of the beholder's tricks at Gate Route.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: beholder -> Beholder

File: QUEST_LV_0100.tsv (hunk 2 above)
Comment (vtange) [inline QUEST_LV_0100.tsv]: oh ok, i'll change it back then

File: QUEST.tsv (hunk 1 above)
Comment (vtange) [inline QUEST.tsv]: ah, np++ keeps suggesting Brit English to me and i finally fell for it

**Comment by Sunaries** [comment]
:+1:

**Comment by vtange** [comment]
:+1:

**Comment by imcgames** [comment]
:+1:

**Pull request merged**